etc/mqtt_source.json

```diff
@@ -0,0 +1,171 @@
+{
+	"author": {
+		"name": "Jiyong Huang",
+		"email": "huangjy@emqx.io",
+		"company": "EMQ Technologies Co., Ltd",
+		"website": "https://www.emqx.io"
+	},
+	"libs": [
+	],
+	"helpUrl": {
```

Comment:
Please add `about` section as for sinks.

```
"about": {
		"trial": false,
		"description": {
			"en_US": "This a sink plugin for InfluxDB, it can be used for saving the analysis data into InfluxDB.",
			"zh_CN": "本插件为 InfluxDB 的持久化插件，可以用于将分析数据存入 InfluxDB 中。"
		}
	},
```